tests/sentry/models/test_activity.py

```diff
@@ -30,6 +29,7 @@ def test_get_activities_for_group_priority_ff_on(self):
         user1 = self.create_user()
         group = event.group
         assert group is not None
+        group.refresh_from_db()
```

Comment:
Do we know why the group has the wrong data without this call? I'm curious if that's indicative of another kind of problem, or if this is just something that we need to remember to do in tests.

Reply:
The latter, I think. My understanding is the `refresh_from_db()` serialized the Enum to an int as expected when we're running in production since the groups won't be in memory after creation.